domains/transfer/Transfer.API/API/Transfer/Api/Controllers/TransferAgreementProposalController.cs

```diff
@@ -150,29 +160,29 @@ public async Task<ActionResult> DeleteTransferAgreementProposal(Guid id)
             return NotFound();
         }
 
-        var user = new UserDescriptor(User);
+        accessDescriptor.AssertAuthorizedToAccessOrganization(organizationId);
```

Comment:
Same as before. I would put this first. SECURITY FIRST?!?!?!?! MAKE SECURITAAAA GRAET AGAIN! :P

Reply:
It was there before. did not change anything